### PR TITLE
anaconda3: Fix setup.exe not found error

### LINKED ISSUE #10833: [Bug]: failed to install anaconda3 due to path error
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

anaconda3

### Expected/Current Behaviour

successful installation

### Steps to Reproduce

```console
scoop install extras/anaconda3
```

### Possible Solution

Modify the path in installer script from `$dir\\setup.exe` to `$dir\\..\\setup.exe` can fix that.

### Scoop and Buckets Version

```console
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
367c77d84 (HEAD -> master, origin/master, origin/HEAD) termius: Update to version 7.58.2
```

### Scoop Config

```console
proxy                 : 127.0.0.1:15524
aria2-enabled         : True
aria2-warning-enabled : False
last_update           : 2023/3/24 14:48:40
scoop_branch          : master
scoop_repo            : https://github.com/ScoopInstaller/Scoop
```

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.3.3
PSEdition                      Core
GitCommitId                    7.3.3
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Additional Softwares

_No response_

### PR DESCRIPTION
**Pull request by silicer**
Closes #10833

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/anaconda3.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
             "Move-Item \"$dir\\setup.exe\" \"$dir\\..\\setup.exe\" | Out-Null",
```
Comment (jasongodev) [inline bucket/anaconda3.json:21]: A recent merged #10830 about 2 days ago was done that added this line. Let's see what is the rationale for moving the setup.exe to another path. If this line is needed then your PR is also needed.

**Review by jasongodev — COMMENTED** [review]
I don't recommend merging until the reason for moving the setup.exe file to another directory is established, see PR #10830.

**Comment by jasongodev** [comment]
For context, miniconda does have the path like this: https://github.com/ScoopInstaller/Extras/blob/14249cdf18a9f8c394faabf7d0814ade9298f326/bucket/miniconda3.json#L24

File: bucket/anaconda3.json (hunk 1 above)
Comment (niheaven) [reply bucket/anaconda3.json:21]: Ref: https://github.com/ScoopInstaller/Extras/issues/10829

**Comment by Lutra-Fs** [comment]
just rename is ok

**Comment by Lutra-Fs** [comment]
Am I rename the installer？

File: bucket/anaconda3.json (hunk 1 above)
Comment (silicer) [reply bucket/anaconda3.json:21]: I don't know why but moving `setup.exe` to the parent directory did solve my installation issue. The `setup.exe` suddenly ends after verifying 100% before. #10829

Comment (Lutra-Fs) [reply bucket/anaconda3.json:21]: I am not sure where we discuss the issue on too many references

**Comment by jasongodev** [comment]
> Am I rename the installer？

I don't think it's necessary. Also, this PR was merged already. So all is well until the next issue.

**Pull request merged**